services/commissioning.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Rovercode Commissioning Service
+After=syslog.target docker.service
+Requires=docker.service
+
+[Service]
+TimeoutStartSec=0
+Restart=always
+User=root
+Group=root
+ExecStartPre=-/usr/bin/docker stop rovercode
+ExecStart=-/usr/bin/docker run --name rovercode-commissioning -v /opt/rovercode/scripts/commission.py:/var/rovercode/scripts/commission.py --entrypoint=/bin/bash rovercode -c 'python /var/rovercode/scripts/commission.py'
```

Comment:
Instead of working with the RaspberryPi's native python, let's assume that our base image has some version of the Rovercode Docker image on it.